core/model/modx/modconnectorresponse.class.php

```diff
@@ -179,7 +179,7 @@ public function outputContent(array $options = array()) {
             if (!empty($_GET['callback'])) {
                 $json = $modx->stripTags($_GET['callback']) . '(' . $json . ')';
             }
-            die($json);
+            die('<textarea>'. $json .'</textarea>');
```

Comment:
I remember you talking about a hack-ish approach, but I didn't quite see this one coming 😬This is going to break interaction with connectors from non-extjs sources, which I know people are doing, so I think we'll have to go back to the drawing board on this one...

Reply:
Yes, I thought this might have far-reaching consequences. This was working back from Ext.data.Connection which provides for this.